aten/src/ATen/native/native_functions.yaml

```diff
@@ -4147,7 +4147,7 @@
   python_module: nn
   dispatch:
     CPU, CUDA: mish_backward
-    CompositeImplicitAutograd: math_mish_backward
+    CompositeExplicitAutograd: math_mish_backward
```

Comment:
So it is not possible to do `CompositeImplicit` + another key anymore?

Reply:
Is `mish_backward` different numerically from the one that would be obtained in the decomp?

Reply:
Just tested it out and it looks like you're right - `torch.allclose` with the defaults thinks they're the same numerically:
```
def f(a, b):
    return torch.ops.aten.mish_backward(a, b)

a = torch.randn(8, dtype=torch.float64)
b = torch.randn(8, dtype=torch.float64)

out1 = f(a, b)
out2 = functionalize(f)(a, b)

print(torch.allclose(out1, out2))

f_traced = make_fx(f)(a, b)
f_no_mutations_traced = make_fx(functionalize(f))(a, b)

print(f_traced.code)
print(f_no_mutations_traced.code)
```
prints:
```
def forward(self, a_1, b_1):
    mish_backward = torch.ops.aten.mish_backward(a_1, b_1);  a_1 = b_1 = None
    return mish_backward




def forward(self, a_1, b_1):
    softplus = torch.ops.aten.softplus(b_1)
    tanh = torch.ops.aten.tanh(softplus);  softplus = None
    sigmoid = torch.ops.aten.sigmoid(b_1)
    mul = torch.ops.aten.mul(tanh, tanh)
    empty_like = torch.ops.aten.empty_like(mul, memory_format = torch.preserve_format)
    fill = torch.ops.aten.fill(empty_like, 1);  empty_like = None
    sub = torch.ops.aten.sub(fill, mul);  fill = mul = None
    mul_1 = torch.ops.aten.mul(b_1, sigmoid);  b_1 = sigmoid = None
    mul_2 = torch.ops.aten.mul(mul_1, sub);  mul_1 = sub = None
    add = torch.ops.aten.add(tanh, mul_2);  tanh = mul_2 = None
    mul_3 = torch.ops.aten.mul(a_1, add);  a_1 = add = None
    return mul_3
```

Would you rather keep the op as `CompositeImplicitAutograd`? I guess the pros/cons are pretty minor:
- pro: the output of your trace will be the same regardless of whether you're running a transform
- con: none? (since CPU/CUDA will treat it as a primitive today, so only e.g. XLA / functorch transforms will actually use the composite.